src/server_manager/web_app/ui_components/outline-manual-server-entry.js

```diff
@@ -127,6 +127,20 @@ Polymer({
           display: none;
         }
       }
+      #gcp-tag {
+        font-weight: 500;
+        letter-spacing: 0.05em;
+        font-size: 10px;
+        margin-bottom: 8px;
+        text-transform: uppercase;
+        color: #4285f4;
```

Comment:
Don't introduce a new color. Use the same color we use in the titles.

Reply:
Done.  (That color was --gcp-blue, but the variable isn't available in this scope.)

Reply:
I still see the blue